rebar.config

```diff
@@ -8,7 +8,7 @@
 {profiles,
  [{test, [{cover_enabled, true}, {cover_opts, [verbose]}, {ct_opts, [{verbose, true}]}]}]}.
 
-{alias, [{test, [compile, format, hank, lint, xref, dialyzer, ct, cover, ex_doc]}]}.
+{alias, [{test, [compile, lint, xref, dialyzer, ct, cover, ex_doc]}]}.
```

Comment:
We should definitely write up a ticket to recover these 2 tools once they incorporate the new katana version we'll create with these changes.